.buildkite/serverless.beats.tests.yml

```diff
@@ -0,0 +1,17 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  DOCKER_REGISTRY: "docker.elastic.co"
+  VAULT_PATH: "kv/ci-shared/observability-ingest/cloud/gcp"
```

Comment:
What is the purpose of this environment variable? I cannot find any references to it in this PR.

Reply:
I think we can delete it. It's not used neither in agent nor beats